### PR TITLE
[UPDATE] Build Version and Dependencies

### PR DESCRIPTION
**Pull request by mwshubham**

### CODE REVIEW

File: CompleteCode/app/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 27
     defaultConfig {
         applicationId "com.rajsuvariya.diffutilsrecyclerview"
         minSdkVersion 16
-        targetSdkVersion 25
+        //noinspection OldTargetApi
+        targetSdkVersion 27
```
Comment (rajsuvariya) [inline CompleteCode/app/build.gradle:9]: Please upgrade to 28

File: CompleteCode/app/build.gradle [hunk 2]
```diff
@@ -21,11 +21,13 @@ android {
...
+
+    /*Testing*/
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation('com.android.support.test.espresso:espresso-core:3.0.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:25.1.1'
-    compile 'com.android.support:design:25.1.1'
-    testCompile 'junit:junit:4.12'
-    compile 'com.android.support:recyclerview-v7:25.1.1'
```
Comment (rajsuvariya) [inline CompleteCode/app/build.gradle:30]: Why are you removing this dependency?

File: CompleteCode/app/build.gradle [hunk 3]
```diff
@@ -21,11 +21,13 @@ android {
...
+
+    /*Android/Support*/
+    implementation 'com.android.support:design:27.1.1'
+
+    /*Testing*/
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation('com.android.support.test.espresso:espresso-core:3.0.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:25.1.1'
```
Comment (rajsuvariya) [inline CompleteCode/app/build.gradle:27]: Why are you removing this dependency?

**Review by rajsuvariya — CHANGES_REQUESTED** [review]
Needs changes as per the comments

File: CompleteCode/app/build.gradle (hunk 3 above)
Comment (mwshubham) [reply CompleteCode/app/build.gradle:27]: design library already contain it

File: CompleteCode/app/build.gradle (hunk 2 above)
Comment (mwshubham) [reply CompleteCode/app/build.gradle:30]: design library already contains it.

File: CompleteCode/app/build.gradle (hunk 1 above)
Comment (mwshubham) [reply CompleteCode/app/build.gradle:9]: I thought at least 28 is ok as of now as for 28 compile sdk. Android has recommended the androidx

**Review by rajsuvariya — APPROVED** [review]

**Pull request merged**